### PR TITLE
Chore: Use new neuron state icons

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Use the new icons for the change to the new neuron detail page.

# Changes

* Upgrade gix-components.
* Change two icons for the NeuronState mapper.
* Set the size of the neuron state icon instead of relying on the default.

# Tests

Only UI changes. No test needed.

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

**Comment by lmuntaner** [comment]
New UI:

Screenshot 2023-07-17 at 11 10 25

**Comment by lmuntaner** [comment]
@peterpeterparker @mstrasinskis please review

**Review by peterpeterparker — COMMENTED** [review]
It's me or the icon is not vertically centered?

253929358-8e0dcaf4-7cb3-49c5-8d70-c9fde538c15c

File: CHANGELOG-Nns-Dapp.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ The NNS Dapp is released through proposals in the Network Nervous System. Theref
 * Improve responsiveness in proposal filter modals.
 * Only Locked neurons are mergeable.
 * Add copy button to sns proposal proposer id.
+* New neuron state icons.
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply CHANGELOG-Nns-Dapp.md:25]: The branch has not yet been merged.

**Comment by lmuntaner** [comment]
> It's me or the icon is not vertically centered?

It looks more like a visual effect.

Screenshot 2023-07-17 at 14 00 27

**Comment by lmuntaner** [comment]
@peterpeterparker to be honest. I also see it weird.

I might look into it but honestly, we'll most likely move to table view for list of neurons that I'm not sure this is worth the effort.

File: CHANGELOG-Nns-Dapp.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply CHANGELOG-Nns-Dapp.md:25]: Ah you will think before merge, good then!

**Pull request merged**